### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by jrgorecki** [jrgorecki/Project#7]
I think all that's left is testing and fixing any weirdness.

**Comment by dougives** [comment]
Yeah, I'm going to make sure all the parts work together tonight and tomorrow morning. All of the cases are covered I think, so any errors should only be the result of me 'misinterpreting' the description :).

**Comment by jrgorecki** [comment]
How is it going? Is everything working smoothly? I noticed an function called "run" that is to be defined in the inter.fs file, but the assignment says nothing about it.
I just assumed it took a string, parsed, checked, and runs it.

**Comment by dougives** [comment]
Sorry, I got held up yesterday, I'm looking at it now.
